frontend/catalyst/compiler.py

```diff
@@ -116,28 +116,33 @@ def get_default_flags(options):
 
         # Discover the LAPACK library provided by scipy & add link against it.
         # Doing this here ensures we will always have the correct library name.
-        lib_name = "openblas"
-        package_name = "scipy_openblas32"
-        path_within_package = "lib"
-        file_extension = ".so" if platform.system() == "Linux" else ".dylib"
-
-        package_spec = importlib.util.find_spec(package_name)
-        package_directory = path.dirname(package_spec.origin)
-        lapack_lib_path = path.join(package_directory, path_within_package)
-
-        search_pattern = path.join(lapack_lib_path, f"lib*{lib_name}*{file_extension}")
-        search_result = glob.glob(search_pattern)
-        if not search_result:
-            raise CompileError(
-                f'Unable to find OpenBLAS library at "{search_pattern}". '
-                "Please ensure that scipy-openblas32 is installed and available via pip."
-            )
+        if platform.system() == "Linux":
+            lib_name = "openblas"
+            package_name = "scipy"
```

Comment:
Does this mean that on Linux we go back to depending on `scipy`, rather than `scipy_openblas32`? If so, does the `requirements` list in setup.py need to be updated as well?

Reply:
I think that is the easiest solution, since the symbols match the expected once (without the `scipy_` prefix from `scipy_openblas32`). I will add it back without a version pin 👍